receiver/elasticsearchreceiver/metadata.yaml

```
@@ -242,6 +242,15 @@ metrics:
       value_type: int
     attributes: [ query_cache_count_type ]
     enabled: true
+  elasticsearch.node.cache.size:
+    description: Total amount of memory, in bytes, used for the query cache across all shards assigned to the node.
```

```suggestion
    description: Total amount of memory used for the query cache across all shards assigned to the node.
```
Since we have a dedicated field for units, typically we leave them out of the description unless necessary.